src/FontManagerMac.mm

```diff
@@ -78,6 +78,11 @@ static int convertWidth(float unit) {
   ResultSet *results = new ResultSet();
   
   for (id m in matches) {
+    NSString *family = (NSString *) CTFontDescriptorCopyAttribute((CTFontDescriptorRef) m, kCTFontFamilyNameAttribute);
```

Comment:
maybe we could do this inside `createFontDescriptor` so it isn't repeated so much. we also need to remember to release the memory for this since this makes a copy.

Reply:
Sorry for the late reply. I will close this and when I have time I will improve it based on your suggestions, thanks.